R/lsmeans.R

```diff
@@ -34,10 +44,42 @@ lscombinations <- function(data, mterms, fix) {
 }
 
 
-#' Title - TODO
-#' 
-#' @param model TODO
-#' @param ... TODO
+#' Least Square Means
+#'
+#' Estimates the least square means from a linear model. This is
+#' essentially where we generate a prediction from the model
+#' by fitting it to some hypothetical observation that is constructed
+#' by averaging the data. See details for more information.
```

Comment:
"This is essentially where we generate a prediction from the model by fitting it to some hypothetical observation that is constructed by averaging the data." ->
"It consists in generating a prediction from the model using an hypothetical observation that is constructed by averaging the data."

Reply:
Done